adam/language_specific/english/english_language_generator.py

```diff
@@ -696,6 +697,10 @@ def _collect_action_modifiers(
                     modifiers.append(
                         (ADVERBIAL_MODIFIER, DependencyTreeToken("down", ADVERB))
                     )
+                elif action.action_type == JUMP:
+                    modifiers.append(
+                        (ADVERBIAL_MODIFIER, DependencyTreeToken("up", ADVERB))
```

Comment:
Should jumping over / in / in front of / behind / beside things be covered in the language generator?

Reply:
Those prepositions are covered in `preposition_for_region_as_goal`; they're not specific to `jump`